src/pretix/presale/views/event.py

```diff
@@ -913,8 +920,49 @@ def post(self, request, *args, **kwargs):
         except:
             raise PermissionDenied(_('Please go back and try again.'))
         else:
-            if 'event_access' not in parentdata:
+            if 'child_session_{}'.format(request.event.pk) not in parentdata:
                 raise PermissionDenied(_('Please go back and try again.'))
 
         request.session['pretix_event_access_{}'.format(request.event.pk)] = parent
         return redirect_to_url(eventreverse(request.event, 'presale:event.index'))
+
+
+class TimemachineForm(forms.Form):
+    now_dt = forms.SplitDateTimeField(
```

Comment:
Can we make that timestamp not required and if left empty, just deactivate time machine? I tried to deactivate time machine by emptying the inputs, then realized there is a deactivate-button and it did not submit. Alternatively instead of allowing empty values (which I think I prefer) would be to have the deactivate-button in its own `<form>`.

Reply:
> allowing empty values

This causes some weird behavior if only one of the fields is empty, where I think an error message should be displayed instead of just disabling (and displaying a "success" message). I'll go with the two forms approach.